### PR TITLE
 fix(sync-server): update docs to reflect new reject method

### PR DESCRIPTION
**Pull request by darahayes**
Related Ticket: https://issues.jboss.org/browse/AEROGEAR-8679

### CODE REVIEW

File: modules/ROOT/pages/_partials/sync/server-offline-and-conflict.adoc [hunk 1]
```diff
@@ -188,17 +186,30 @@ const { conflictHandler, strategies } = require('@aerogear/voyager-conflicts')
 
 When there is no conflict, `conflictHandler.nextState(clientData)` is called and the data is persisted. When a conflict occurs, the following happens.
 
-* `conflictHandler.resolveOnServer` is called with the `serverWins` strategy. In this case, the `resolvedState` will be the current server record with an updated version number. *The resolvedState must be persisted as the version is updated.*
+* `conflictHandler.resolveOnServer` is called with the `clientWins` strategy. In this case, the `resolvedState` will be the new data provided by the client. The newly `resolvedState` should be persisted.
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/server-offline-and-conflict.adoc]: can we avoid 'should'?
How about 'You need to persist the `resolvedState` data.' or similar?

Comment (finp) [reply modules/ROOT/pages/_partials/sync/server-offline-and-conflict.adoc]: @darahayes  merged as i posted review, just wondering tho, is my understanding correct in the comment I made?

Comment (darahayes) [reply modules/ROOT/pages/_partials/sync/server-offline-and-conflict.adoc]: @finp apologies for that. Your understanding is definitely correct. How about I just switch the word `should` with `must`. **The newly `resolvedState` must be persisted**. WDYT?

**Comment by finp** [comment]
@darahayes  still very passive, I think what you want the user to do is write code to persist the data, right?
If so, let's tell them that, and avoid ambiguity.

**Pull request merged**